### PR TITLE
Address Making "Performance Input Method" Input Field for EvaporativeFluidCooler:SingleSpeed a required-field

### LINKED ISSUE #9585: Making "Performance Input Method" Input Field for EvaporativeFluidCooler:SingleSpeed a required-field
Issue overview
--------------
The "Performance Input Method" input field for EvaporativeFluidCooler:SingleSpeed is a required-field per the source code but the idd does not label it as a required field. See comments in issue #9262. 

### Details
Some additional details for this issue (if relevant):
 - Platform (Operating system, version)
 - Version of EnergyPlus (if using an intermediate build, include SHA)
 - Unmethours link or helpdesk ticket number

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Defect file added (list location of defect file here)
 - [ ] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

### PR DESCRIPTION
**Pull request by Nigusse**
Pull request overview
---------------------
 - Fixes #9585
 - Performance Input Method for single speed evap fluid cooler is a required-field. This change just does that.

**NOTE: ENHANCEMENTS MUST FOLLOW A SUBMISSION PROCESS INCLUDING A FEATURE PROPOSAL AND DESIGN DOCUMENT PRIOR TO SUBMITTING CODE**

### Pull Request Author
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [ ] Label the PR with at least one of: Defect, Refactoring, NewFeature, Performance, and/or DoNoPublish
 - [ ] Pull requests that impact EnergyPlus code must also include unit tests to cover enhancement or defect repair
 - [ ] Author should provide a "walkthrough" of relevant code changes using a GitHub code review comment process
 - [ ] If any diffs are expected, author must demonstrate they are justified using plots and descriptions
 - [ ] If changes fix a defect, the fix should be demonstrated in plots and descriptions
 - [ ] If any defect files are updated to a more recent version, upload new versions here or on DevSupport
 - [ ] If IDD requires transition, transition source, rules, ExpandObjects, and IDFs must be updated, and add IDDChange label
 - [ ] If structural output changes, add to output rules file and add OutputChange label
 - [ ] If adding/removing any LaTeX docs or figures, update that document's CMakeLists file dependencies

### Reviewer
This will not be exhaustively relevant to every PR.
 - [ ] Perform a Code Review on GitHub
 - [ ] If branch is behind develop, merge develop and build locally to check for side effects of the merge
 - [ ] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] If feature, test running new feature, try creative ways to break it
 - [ ] CI status: all green or justified
 - [ ] Check that performance is not impacted (CI Linux results include performance check)
 - [ ] Run Unit Test(s) locally
 - [ ] Check any new function arguments for performance impacts
 - [ ] Verify IDF naming conventions and styles, memos and notes and defaults
 - [ ] If new idf included, locally check the err file and other outputs

### CODE REVIEW

File: idd/Energy+.idd.in [hunk 1]
```diff
@@ -75858,6 +75858,7 @@ EvaporativeFluidCooler:SingleSpeed,
        \minimum> 0.0
        \ip-units gal/min
   A4 , \field Performance Input Method
+       \required-field
```
Comment (Nigusse) [inline idd/Energy+.idd.in:75861]: Per the source code this input field is a required but it has been missing in the IDD. This change just labels it as required-field per the design.

**Comment by Nigusse** [comment]
I ran the unit test suite locally and all passed. Also ran the `EvaporativeFluidCooler.idf` successfully and I was not expecting any issue. This is a hot fix without a unit test.

**Comment by Nigusse** [comment]
@mjwitte this PR is ready for review.

**Review by mjwitte — APPROVED** [review]
Last CI report was all green. Merged in develop and unit tests pass locally. Merging. Thanks @Nigusse

**Pull request merged**